docs/docs/data-sources/Overview.md

```diff
@@ -0,0 +1,33 @@
+---
+sidebar_position: 1
+---
+
+## Introduction to Data Sources
+
+In this section, we'll explore the concept of data sources within ALGOMO. Data sources play a crucial role in providing information for bots to answer questions. Think of them as the building blocks of knowledge that make up your bot. These sources can take various forms, such as PDF documents, structured Question and Answers (intents), website searches (via Bing or index versions), and more.
+
+Understanding how data sources work is fundamental to configuring your bot effectively. In the following sections, we will delve into the different types of data sources and how they are integrated into the system's workflow.
+
+## Types of Data Sources
+
+There are three primary types of data sources within the system:
+
+- [**Intents**](./Types/Intents.md)
+- [**Documents**](./Types/Documents.md)
+- [**Website**](./Types/Website.md)
+- [**Notion**](./Types/Notion.md)
+
+Each of these data source types follows a distinct lifecycle and serves a unique purpose in the system. Let's explore each of them in greater detail.
```

Comment:
`Let's explore each of them in greater detail.` 
to
`Click on the links above to explore how to create your data sources.`